### DISCUSSION TITLE
multiple language support

### DISCUSSION THREAD

**Discussion by cn-lichaofeng** [flowsforapex/apex-flowsforapex#678] [Q&A] [answered]
hi all,
anyone can explain me where to replace the inexact text of BPMN label?
thanks,
regards

**Comment by rallen2010** [comment, marked as answer]
Do we have some bad translation strings that need correcting?  I'm sure we do!  

Please could you let us know any bad translations ( version, language, current bad text, suggested better text).  Or, even better, if you would like to volunteer to proof check the file.   I assume this is simplified Chinese?  Happy for you to contribute to help the project for this.

The current product architecture means that translation files in the modeler are fairly heavily bundled & not easily changed without us having to re-release product.

**Reply by cn-lichaofeng** [reply]
thanks. yes, I am care about simplified Chinsese now.
I have received the translation json file, and will send it back to develope teams after I made the necessary corrections that I believe should be made

**Reply by rallen2010** [reply]
Hi Chao Feng,  We are getting ready to finalise release 24.1 in the next few weeks.  Could you send me any corrections you have to the simplified Chinese translations - Richard@flowquest.net

**Reply by rallen2010** [reply]
Your changes will be in 24.1.  Anybody else finding bad translations, please let us know & we will try to fix in the next release.
